.travis.yml

```diff
@@ -24,10 +24,11 @@ matrix:
       script: ./ci/build-linux-bazel.sh
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Debug VERBOSE=1
+      install: ./ci/install-linux.sh && ./ci/log-config.sh
+      script: ./ci/build-linux-autotools.sh
     - os: linux
       compiler: gcc
-      env: BUILD_TYPE=Release VERBOSE=1 CXX_FLAGS=-std=c++11
+      env: BUILD_TYPE=Debug VERBOSE=1 CXX_FLAGS=-std=c++11
```

Comment:
@coryan not sure why we are changing Release to Debug and how it is related to the running of automake from the top level dir?

Reply:
That is just to keep the number of builds under control.  The autotools build already builds optimized Linux (Ubuntu) with g++, why have another build do the exact same thing?

Reply:
Happy to revert this change if you prefer.

Reply:
No problem, we can leave them this way, thank you